src/hooks/useForm.js

```diff
@@ -9,10 +9,7 @@ const useForm = (
 
   const handleSubmit = useCallback(() => {
     const newErrors = validator(values) || {};
-    const valid =
-      Object.values(newErrors)
-        .flat()
-        .filter(error => !!error).length === 0;
+    const valid = !Object.values(newErrors).flat().length;
```

Comment:
Don't you need to still filter out falsy values before checking `.length`?

Reply:
nope, since the errors object omits the keys that don't have errors